src/host/ncp_spinel.hpp

```diff
@@ -416,14 +444,6 @@ class NcpSpinel : public InfraIf::Dependencies
                                   uint16_t            &aLocalPort);
     otError SendDnssdResult(otPlatDnssdRequestId aRequestId, const std::vector<uint8_t> &aCallbackData, otError aError);
 
-    otbrError SetInfraIf(uint32_t                       aInfraIfIndex,
-                         bool                           aIsRunning,
-                         const std::vector<Ip6Address> &aIp6Addresses) override;
-    otbrError HandleIcmp6Nd(uint32_t          aInfraIfIndex,
-                            const Ip6Address &aIp6Address,
-                            const uint8_t    *aData,
-                            uint16_t          aDataLen) override;
```

Comment:
Have you deleted the implementations?

Reply:
The two methods are changed to public methods to be called by NcpHost (and not override). But the implementation is still needed.